numpy/distutils/ccompiler.py

```diff
@@ -703,14 +703,17 @@ def CCompiler_cxx_compiler(self):
                             "Intel C Compiler for 32-bit applications on Windows")
 compiler_class['intelemw'] = ('intelccompiler', 'IntelEM64TCCompilerW',
                               "Intel C Compiler for 64-bit applications on Windows")
+compiler_class['clang-cl'] = ('clang_cl', 'ClangCL',
+                              "Clang CL Compiler for 64-bit Windows")
 compiler_class['pathcc'] = ('pathccompiler', 'PathScaleCCompiler',
                             "PathScale Compiler for SiCortex-based applications")
 ccompiler._default_compilers += (('linux.*', 'intel'),
                                  ('linux.*', 'intele'),
                                  ('linux.*', 'intelem'),
                                  ('linux.*', 'pathcc'),
                                  ('nt', 'intelw'),
-                                 ('nt', 'intelemw'))
+                                 ('nt', 'intelemw'),
+                                 ('nt', 'clangcl'))
```

Comment:
This could be future-proofed (at least future-improved) by putting the closing paretheses on its own line
```
('nt', 'intelemw'),
('nt', 'clangcl'),
)
```

Reply:
Fixed.